pyfar/dsp/filter/audiofilter.py

```diff
@@ -279,30 +283,96 @@ def high_shelve(signal, frequency, gain, order, shelve_type='I',
 blob/master/filter_design/audiofilter.py
     """
 
-    output = _shelve(
-        signal, frequency, gain, order, shelve_type, sampling_rate, 'high')
+    warnings.simplefilter('always', PyfarDeprecationWarning)
```

Comment:
is there a specific reason for that?

Reply:
According to the PyfarDeprecationWarning doc, we do not need to use this.

Reply:
Ah, no we don't need it, thanks. It's a leftover